tkg/constants/clusterclass_mapping.go

```diff
@@ -245,6 +245,10 @@ var ClusterAttributesToLegacyVariablesMapVsphere = map[string]string{
 	TopologyWorkersMachineDeploymentsName2:          "",
 	TopologyWorkersMachineDeploymentsReplicas2:      ConfigVariableWorkerMachineCount2,
 	TopologyWorkersMachineDeploymentsFailureDomain2: ConfigVariableVsphereAz2,
+
+	"spec.topology.variables.customTDNFRepository.ip":          ConfigVariableCustomTDNFRepositoryIP,
+	"spec.topology.variables.customTDNFRepository.fqdn":        ConfigVariableCustomTDNFRepositoryFQDN,
```

Comment:
The mapping consts are for reusing legacy variable validation. Should we perform format check on them?

Reply:
This is just a hack that is temporarily provided to TCA. Don't wanna add more time to that

Reply:
ALos if I don't add it here, when user use two steps to create workload cluster, it will fail.